Tasks/AppCenterDistributeV1/task.json

```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 1,
         "Minor": 234,
```

Comment:
Could you please bump the version of the tasks to 235 since the sprint is almost over?

Reply:
Done